cmd/commands/set.go

```diff
@@ -0,0 +1,61 @@
+package command
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/red-hat-storage/odf-cli/pkg/set"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// SetCmd represents the set command
+var SetCmd = &cobra.Command{
+	Use:                "set",
+	Short:              "Call subcommands like 'recover-profile'",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+}
+
+var validArgs = []string{"high_client_ops", "high_recovery_ops", "balanced"}
+
+var recoverProfile = &cobra.Command{
+	Use:                "recover-profile",
+	Short:              "Update the mclock scheduler profile value `high_client_ops, high_recovery_ops, balanced` used during recovery for single osd or all the osds",
```

Comment:
Do we really need the flexibility to set the recovery profile for a single OSD? I thought the downstream cli would just need to set a single global setting so all OSDs are affected. Unless support requested that feature, I would vote to remove that option.
```suggestion
	Short:              "Set the recovery profile to favor new IO, recovery, or balanced mode with options high_client_ops, high_recovery_ops, or balanced",
```

Reply:
I don't have opinion here, I found the option in the ceph docs so I added it but I'm okay to remove the option to pass osd.